lara-typescript/src/lib/plugins.spec.ts

```diff
@@ -18,11 +20,12 @@ describe("Plugins", () => {
 
   describe("registerPlugin", () => {
     it ("should let plugin to register itself only once", () => {
-      const pluginConstructor = jest.fn();
+      const runtimeClass = jest.fn();
+      const authoringClass = jest.fn();
       setNextPluginLabel("test");
-      expect(registerPlugin("*DEPRECATED*", pluginConstructor)).toEqual(true);
+      expect(registerPlugin({runtimeClass, authoringClass})).toEqual(true);
       setNextPluginLabel("test");
-      expect(registerPlugin("*DEPRECATED*", pluginConstructor)).toEqual(false);
+      expect(registerPlugin({runtimeClass, authoringClass})).toEqual(false);
```

Comment:
Since authoringClass is optional now. It'd be good to have a test for that.

Reply:
:+1: I will add.

Reply:
Done.